docs/development.md

```diff
@@ -1,29 +1,5 @@
 # Development
 
-## Running the latest version of the Language Server in the Visual Studio Code Extension
```

Comment:
Was already moved to https://github.com/onflow/cadence-tools/tree/master/languageserver#development before